internal/splunkconfig/config/indexname.go

```diff
@@ -32,7 +32,7 @@ func (indexName IndexName) validate() error {
 	kvstoreRegex := regexp.MustCompile("kvstore")
 
 	// NOTE: we allow leading underscores, because not all Index objects are new indexes, some describe defaults
-	generalRegex := regexp.MustCompile("^[a-z0-9_][a-z0-9_-]+$")
+	generalRegex := regexp.MustCompile("^[a-z0-9_][a-z0-9_-]*$")
```

Comment:
I changes this while I was in here, because I'm unaware of a single-character index name being invalid.